internal/checker/checker.go

```diff
@@ -11521,7 +11533,10 @@ func (c *Checker) checkBinaryLikeExpression(left *ast.Node, operatorToken *ast.N
 				ast.KindGreaterThanGreaterThanGreaterThanEqualsToken:
 				rhsEval := c.evaluate(right, right)
 				if numValue, ok := rhsEval.Value.(jsnum.Number); ok && numValue.Abs() >= 32 {
-					c.errorOrSuggestion(ast.IsEnumMember(ast.WalkUpParenthesizedExpressions(right.Parent.Parent)), errorNode, diagnostics.This_operation_can_be_simplified_This_shift_is_identical_to_0_1_2, scanner.GetTextOfNode(left), scanner.TokenToString(operator), (numValue / 32).Floor())
+					// Adding 0 removes sign from -0
+					shiftMod32 := math.Mod(float64(numValue), 32) + 0
```

Comment:
Probably just `numValue.Remainder(32)`? I'd rather not rely on the `math` package outside of `jsnum`; I eliminated it everywhere except for the max int consts.

Reply:
Yes, I'll switch to `Number.Remainder`. Which by the way has a bug in it that I'll also fix.